### ISSUE TITLE
Manual level setting

### ISSUE THREAD

**Issue by BrechtDeMan** [BrechtDeMan/WebAudioEvaluationTool#173]
Either as a calibration stage for the rest of the test, or as a test in itself - e.g. evaluation of loudness standards, it may be desired to set the levels of samples (i.e. modify the gain attributes of the audio elements) in preparation for a test. 

This could be done by an expert panel, or by each subject individually before each test. 

Could be a horizontal or vertical slider interface, with virtually unlimited number of fragments at the same time (perhaps always all the audio elements in the test). 

Quite specific use case and somewhat of a stretch goal, but has been known to happen. This ventures into the Pandora's box that is 'method of adjustment' tests, but fairly basic.

**Comment by nickjillings** [comment]
I see this really being split into categories:

1. Adjustment when inside the test creator. This is something that we played with in the very first one, however I removed the Web Audio stuff for simplicity for now. If it is for a panel / dev, then this should be done in the test_creator

2. Method of adjustment tests. Slider interface (can actually be any of the continuous interfaces) which map axis onto the gain controls.

**Comment by BrechtDeMan** [comment]
I don't think it needs to be in the test creator (except of course the field for gain). 

Simple MOA tests like this would be nice, especially if they can be chained together with other tests. And if that gain can be used by subsequent tests? 

Not sure if I understood you correctly.
